tests/fixtures/language_container_fixture.py

```diff
@@ -32,18 +32,23 @@ def export_slc(flavor_path: Path) -> ExportInfo:
 @pytest.fixture(scope="session")
 def upload_slc(itde: TestConfig, flavor_path: Path, export_slc: ExportInfo) -> Path:
     cleanup_images()
-    container_bucketfs_location = \
-        BucketFSFactory().create_bucketfs_location(
-            url=f"{itde.bucketfs.url}/{bucketfs_params.bucket}/{bucketfs_params.path_in_bucket};{bucketfs_params.name}",
-            user=itde.bucketfs.username,
-            pwd=itde.bucketfs.password)
+    container_bucketfs_location = create_bucketfs_location(
+        path_in_bucket=bucketfs_params.path_in_bucket,
+        bucketfs_name=bucketfs_params.name,
+        bucketfs_url=itde.bucketfs.url,
+        bucketfs_user=itde.bucketfs.username,
+        bucketfs_password=itde.bucketfs.password,
+        bucket=bucketfs_params.bucket,
+        use_ssl_cert_validation=False)
     container_file_path = Path(export_slc.cache_file)
+    container_file_bfs_path = container_bucketfs_location / container_file_path.name
+
     with open(export_slc.cache_file, "rb") as fileobj:
-        container_bucketfs_location.upload_fileobj_to_bucketfs(
-            fileobj,
-            container_file_path.name)
+        container_file_bfs_path.write(fileobj)
```

Comment:
The following would be an alternative to the with block:
```python
container_file_bfs_path.write(
    container_file_path.read_bytes()
)
```

Reply:
But that would mean reading the whole file in memory first. The file is quite big, so it's better to use the file object in this instance.